flume/doc/installation/rpm_install.md

```diff
@@ -1 +1,16 @@
-TBD
+#Installing Cygnus via RPM
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-connectors/commit/80d007dce480ab9fd869e220bbe585209ffc5681